roles/image/templates/image.conf

```diff
@@ -52,7 +52,7 @@ server {
     # First attempt to serve request as file, then
     # as directory, then fall back to displaying a 404.
     index  index.html index.htm;
-    try_files $uri $uri/ =404;
+    try_files $uri $uri/ /index.html;
```

Comment:
This means that requesting a url like https://www.wp5image.eu/a/fake/url will never return a 404 page but https://www.wp5image.eu/ instead. Is this required in data-portal application? could you provide some examples? Currently in [fail2ban role](https://github.com/cnr-ibba/IMAGE-ansible/tree/master/roles/fail2ban) we track users that are trying to scraping our website against pages that don't exist: with this commit they will be redirected to image [home page](https://www.wp5image.eu/) when requesting a non existent url.

Reply:
And never be banned